### DISCUSSION TITLE
No Copilot access

### DISCUSSION THREAD

**Discussion by dsewnr** [community/community#6872] [Copilot Conversations] [answered]
I installed `copilot.vim` but don't have Copilot access. I'm just saying.

**Comment by TarunavBA** [comment, marked as answer]
Did you first get access to `github copilot`? Like did you get the email? If u didn't then u dont have access to copilot.

**Reply by mtderosier** [reply]
How long does it take on average to be given access to copilot chat?

**Comment by Kobester-nz** [comment]
unable to login to github when using the copilot
